plsql/ExcelGen.pkb

```diff
@@ -1579,7 +1584,10 @@ create or replace package body ExcelGen is
         meta.visibleColumnSet(i) := meta.columnList(i).colRef;
       end if;
     end loop;
-    
+  exception when others then
+    debug('cursorNumber: '||to_char(meta.cursorNumber));
+    debug('queryString: '||meta.queryString);
+    raise;
```

Comment:
I'm not OK with this RAISE statement as we lose the line number where the error really occurred.
Actually, I'm not fond of any exception handler here, just let the exception propagate to the caller, and put debugs inside the main PL/SQL block.